yellowbrick/features/pcoords.py

```diff
@@ -149,6 +152,15 @@ class ParallelCoordinates(DataVisualizer):
         If int, specifies the maximum number of samples to display.
         If float, specifies a fraction between 0 and 1 to display.
 
+    random_state : int, RandomState instance or None
```

Comment:
Awesome +100 for this - thank you so much for thinking to add this in!

Reply:
:+1: